manifests/aws-cloud-controller-manager-daemonset.yaml

```diff
@@ -28,7 +28,7 @@ spec:
       serviceAccountName: cloud-controller-manager
       containers:
         - name: aws-cloud-controller-manager
-          image: gcr.io/k8s-staging-provider-aws/cloud-controller-manager:v1.18.0-alpha.1
+          image: gcr.io/k8s-staging-provider-aws/cloud-controller-manager:v1.19.1-alpha.1
```

Comment:
I don't think we require patch versions to match, so this should be v1.19.0-alpha.1?

Reply:
Also, how did you build this without https://github.com/kubernetes/cloud-provider-aws/pull/136, or we didn't build this yet?

Reply:
cc @nicolehanjing